example/src/spa/view/header/header-view.js

```diff
@@ -0,0 +1,83 @@
+import './header.css';
+// eslint-disable-next-line no-unused-vars
+import Router from '../../router/router';
+import { Pages } from '../../router/pages';
+import View from '../../../classes/view/view';
+import ElementBuilder from '../../../classes/util/element/element-builder';
+
+const CssClasses = {
+    HEADER: 'header',
+    NAV: 'nav',
+    ITEM: 'nav-item',
+    ITEM_SELECTED: 'nav-item__selected',
+};
+const NamePages = {
+    INDEX: 'Главная',
+    SHOWCASE: 'Карточки',
+    ABOUT: 'О нас',
+};
+
+export default class HeaderView extends View {
+    /**
+     * @param {Router} router
+     */
+    constructor(router) {
+        super();
+
+        this.headerLinkElements = new Map();
+        this.htmlElement = this.createView(router);
+    }
+
+    /**
+     * @param {Router} router
+     */
+    createView(router) {
+        const headerBuilder = new ElementBuilder('header');
+
+        const navBuilder = new ElementBuilder('nav');
+        navBuilder.setCssClasses([CssClasses.NAV]);
+
+        Object.keys(NamePages).forEach((key) => {
+            const callback = () => router.navigate(Pages[key]);
+
+            const builder = new ElementBuilder('a');
+            builder
+                .setCssClasses([CssClasses.ITEM])
+                .setTextContent(NamePages[key])
+                .setClickCallback(this.linkClickHandler.bind(this, callback));
```

Comment:
We could pass an arrow function instead of using `bind`.
`setClickCallback(() => {...})`